pkg/webhook/validator_test.go

```diff
@@ -2239,19 +2219,7 @@ func TestValidatePodSpecNonDefaultNamespace(t *testing.T) {
 				},
 			},
 		),
-		want: func() *apis.FieldError {
```

Comment:
Just wondering if there's any point in keeping these tests around?

Reply:
I think it is fine to keep them so we can exercise these all these fields in case there are errors when using go-containerregistry in the future.